app/controllers/api/v1/developers_controller.rb

```diff
@@ -0,0 +1,33 @@
+# frozen_string_literal: true
+
+module Api::V1
+  class DevelopersController < Api:: BaseController
+    before_action :set_developer, only: %i[show]
+
+    def index
```

Comment:
`index` is not a scope this time.
But I wonder if all of the includes are needed.
Can you make sure?